src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubAttribute.cs

```diff
@@ -10,8 +10,9 @@ namespace Microsoft.Azure.WebJobs.ServiceBus
     /// Setup an 'output' binding to an EventHub. This can be any output type compatible with an IAsyncCollector.
     /// </summary>
     [AttributeUsage(AttributeTargets.Parameter)]
+    [ConnectionProvider(typeof(ServiceBusAccountAttribute))]
```

Comment:
Fine with me to disallow method/class lvl - though EventHubs are service bus entities, so it's an option.